TestON/tests/FuncIntent/Dependency/FuncIntentFunction.py

```diff
@@ -0,0 +1,543 @@
+"""
+    Wrapper functions for FuncIntent
+    This functions include Onosclidriver and Mininetclidriver driver functions
+"""
+def __init__( self ):
+    self.default = ''
+
+def hostIntent( main,
+                name="",
+                host1="",
+                host2="",
+                host1Id="",
+                host2Id="",
+                mac1="",
+                mac2="",
+                vlan1="-1",
+                vlan2="-1",
+                sw1="",
+                sw2="",
+                expectedLink=0 ):
+    """
+        Add host intents
+    """
+    import time
+
+    # Assert variables
+    assert main, "There is no main variable"
+    assert name, "variable name is empty"
+    assert host1 and host2, "You must specify hosts"
+
+    global itemName
+    itemName = name
+    h1Name = host1
+    h2Name = host2
+    h1Id = host1Id
+    h2Id = host2Id
+    h1Mac = mac1
+    h2Mac = mac2
+    vlan1 = vlan1
+    vlan2 = vlan2
+    intentsId = []
+    stepResult = main.TRUE
+    pingResult = main.TRUE
+    intentResult = main.TRUE
+    flowResult = main.TRUE
+    topoResult = main.TRUE
+    linkDownResult = main.TRUE
+    linkUpResult = main.TRUE
+
+    if main.hostsData:
+        if not h1Mac:
+            h1Mac = main.hostsData[ h1Name ][ 'mac' ]
+        if not h2Mac:
+            h2Mac = main.hostsData[ h2Name ][ 'mac' ]
+        print '1', main.hostsData[ h1Name ]
```

Comment:
Your right I think I remove it after the commit.

On Tue, May 19, 2015 at 11:05 AM, Jon Hall <notifications@github.com> wrote:

> In TestON/tests/FuncIntent/Dependency/FuncIntentFunction.py
> <https://github.com/opennetworkinglab/ONLabTest/pull/117#discussion_r30628001>
> :
>
> > +    vlan2 = vlan2
> > +    intentsId = []
> > +    stepResult = main.TRUE
> > +    pingResult = main.TRUE
> > +    intentResult = main.TRUE
> > +    flowResult = main.TRUE
> > +    topoResult = main.TRUE
> > +    linkDownResult = main.TRUE
> > +    linkUpResult = main.TRUE
> > +
> > +    if main.hostsData:
> > +        if not h1Mac:
> > +            h1Mac = main.hostsData[ h1Name ][ 'mac' ]
> > +        if not h2Mac:
> > +            h2Mac = main.hostsData[ h2Name ][ 'mac' ]
> > +        print '1', main.hostsData[ h1Name ]
>
> Can you either use main.log.* for these print statements or if they are
> debug messages remove them before committing?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/opennetworkinglab/ONLabTest/pull/117/files#r30628001>.
>